api/context.ts

```diff
@@ -1,11 +1,12 @@
 import { PrismaClient } from '@prisma/client'
+import { db } from './db'
 
 export interface Context {
   db: PrismaClient
 }
 
 export function createContext(): Context {
   return {
-    db: new PrismaClient(),
+    db,
```

Comment:
Is there a reason we're passing a function as the context instead of just passing a regular Object?

Reply:
Because a function is what Apollo Server expects. It is called on every request. Usually work is done here to expose request scoped details to resolvers, like current user.

This needs to be explained better in the tutorial.

Reply:
>  Usually work is done here to expose request scoped details to resolvers, like current user.

Yup, I've actually done that in an app I'm building. 

> Because a function is what Apollo Server expects.

Apollo Server actually expects either a [function or a plain javascript object](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#context). It's not really a big deal, but I thought it might've been simpler just passing a `context` object in this case.

Reply:
Good to know, in the spirit of the tutorial that incrementally builds up on an as-needed basis I would happily accept a PR to make this change!

Reply:
Done at graphql-nexus/schema#718!